core/services/chainlink/application.go

```diff
@@ -698,8 +661,15 @@ func (app *ChainlinkApplication) SessionORM() sessions.ORM {
 	return app.sessionORM
 }
 
+// TODO remove this all together
 func (app *ChainlinkApplication) EVMORM() evmtypes.Configs {
-	return app.Chains.EVM.Configs()
+
+	//TODO
+	// this is close, but the evm interface is slightly different than the
+	// chainset/loop relayer interface wrt chain status.
+	// either write an adapter or change/simplify the evm interface
+	// app.relayers.List(FilterByType(relay.EVM))
+	return nil
```

Comment:
Does this need to be fixed in this PR?

Reply:
very good catch. thank you. it's terrifying that this wasn't caught in any test.

i was trying to flush out a nil ptr and forgot about this.

the orm is barely used in prod, but really deeply embedded in the test/mock code.

spent 15 minutes fixing prod only to spend an hour on the test code and get stuck.

will have to tackle this next week, i don't think i want to rm the interface but i do think that i want to change the implementation. and save the removal for a followup